.woodpecker/docker.yml

```diff
@@ -79,6 +77,8 @@ steps:
       dockerfile: docker/Dockerfile.server.multiarch
       platforms: *platforms_preview
       tag: pull_${CI_COMMIT_PULL_REQUEST}
+      logins: *publish_logins
+      dry_run: true
```

Comment:
I second that. It's convenient though for some non-expert people which don't want to clone and push a PR image.

Maybe it could be somehow conditioned on a PR label? There is `CI_COMMIT_PULL_REQUEST_LABELS` and we could add a "when" condition to the step.